### ISSUE TITLE
Reword the first checkbox option in the Other tab

### ISSUE THREAD

**Issue by redactedscribe** [mtkennerly/ludusavi#122]
The first checkbox option in the Other tab beginning with "In backups, exclude save locations..." has always been confusing to me. I've read it at least three times on three separate occasions and I still don't understand what it's meant to control. Specifically, what is "confirmed on another operating system" mean? Could this paragraph be reworded or simplified?

Thanks!

Ludusavi 0.12.1

**Comment by mtkennerly** [comment]
In the manifest, save locations can be marked with a known OS. For example, if a path is flagged with:

```yaml
when:
  - os: windows
  - os: linux
```

...then the save location is confirmed for Windows and Linux. It may or may not be valid for Mac.

With the above flags, if you're on Mac and check the checkbox, then Ludusavi will skip the save location, because it hasn't explicitly been confirmed for your OS.

Would this be more straightforward?

> Only back up save locations that have been confirmed for this operating system.

**Comment by redactedscribe** [comment]
> Only back up save locations that have been confirmed for this operating system.

I still think "been confirmed" should be replaced with something more explicit and informative. Also a second sentence explaining why you may want to enable it: What does it avoid? Why does it help? Does it speed things up? Is there a reduction in erroneous/wasteful backups? There's probably a good reason why it's disabled by default, but I'm still not getting why I may want to enable it.

**Comment by mtkennerly** [comment]
Hmm, do you have any suggestions? The current version seems clear to me, so it's hard to think of what would make it easier to understand.

In practical terms, here's an example where the distinction matters: https://www.pcgamingwiki.com/wiki/Dead_Cells#Game_data . This game has save locations listed for Windows and Linux, but nothing listed for Mac. In this case, I would say the Windows and Linux save locations are confirmed. There is no confirmed save location for Mac, but since the Windows and Linux paths are the same, it's very likely that the Mac location is the same too.

* A Mac user may want to leave the checkbox unchecked, because then Ludusavi can try checking if the Windows or Linux save locations happen to be valid for Mac.
* A Windows user may want to uncheck the checkbox, because most games have Windows saves listed already. Every extra check adds a little time to the scan and increases the chance of false positives, so a Windows user may not feel it adds enough value for them when the wiki already lists confirmed Windows paths for so many games.

> <img width="642" alt="image" src="https://user-images.githubusercontent.com/4934192/188264482-bfdcc2b6-4281-42d3-bd3e-0965f19f5cc0.png">

There are also some games where the "system" column is just Steam. In that case, we don't have any OS information, so the checkbox doesn't apply. Basically:

* Checkbox on = only check save locations that are explicitly marked with your OS or save locations that don't have any OS information
* Checkbox off = check all known save locations, regardless of OS

**Comment by redactedscribe** [comment]
Thank you for the detailed explanation. I think I understand fully now.

> A Mac user may want to leave the checkbox unchecked, because then Ludusavi can try checking if the Windows or Linux save locations happen to be valid for Mac.

I think you meant _checked_, not unchecked.

Here's my suggestion (the wording or code may need to be adjusted, but the way I've written it seems clearest to me and hopefully others):

> Only [scan|preview] and back up save locations defined for the operating system currently in use. If you are using Windows, [enabling|checking] this will reduce [scan|preview] times at the minor risk of some save locations being undetected and therefore not being backed up (minor because the vast majority of games have Windows save locations defined on PCGamingWiki). If however you are using a less popular gaming operating system, macOS for example, you may instead want to leave this option [disabled|unchecked] to allow checking for save locations defined for other operating systems because not all games have a macOS save location defined (note that doing so may detect more save locations but at the potential cost of introducing false positives). If a game you want to back up isn't being detected, it is better to update the game's PCGamingWiki page with its save location for your current operating system than to permanently disable this option.

It's a bit too wordy for an option's name, but I think including a proper explanation is justified. This could be tackled by putting everything but the first sentence into a tooltip, or provide some clickable "?" button/text to reveal the rest. I think mention of PCGamingWiki is necessary because it may not be clear to a user where these save locations are defined (or which are confirmed as you put it).

What do you think of the last sentence in my suggestion? In general, I think it should be discouraged to disable the option (other than temporarily) because its benefits are only for an individual with the potential for more false positives, whereas contributing to PCGW benefits all and should keep backups reliable. I realise the option is disabled by default, but maybe it shouldn't be? Or maybe only disabled by default for OSes other than Windows?

**Comment by mtkennerly** [comment]
I'm actually starting to think I should just remove the option. On Windows, if I enable it, the only difference is that Don't Starve skips two txt log files; everything else was identical, and the scan time was only a fraction of a second faster. So if you're on Windows, the extra scan locations don't really hurt anything, and if you're not on Windows, you almost surely want to leave the option unchecked.

> I think you meant *checked*, not unchecked.

Unchecked = don't exclude = include more 🙃 

> What do you think of the last sentence in my suggestion? In general, I think it should be discouraged to disable the option (other than temporarily) because its benefits are only for an individual with the potential for more false positives

I agree with you, but I also don't think the typical user would be interested in modifying PCGW. The same advice goes for the new file toggles.
